crates/holochain_sqlite/src/db.rs

```diff
@@ -330,14 +330,30 @@ impl<Kind: DbKindT + Send + Sync + 'static> DbWrite<Kind> {
         Self::new(None, kind, DbSyncLevel::default())
     }
 
-    /// Remove the db and directory
-    #[deprecated = "is this used?"]
-    pub async fn remove(self) -> DatabaseResult<()> {
-        if let Some(parent) = self.0.path.parent() {
-            std::fs::remove_dir_all(parent)?;
-        }
-        Ok(())
-    }
+    // /// Delete the db files
```

Comment:
Why is there this commented function added?

Reply:
:point_up: